src/_includes/video.njk

```diff
@@ -1,3 +1,5 @@
+{# Any changes made here, please also do the same in richTextToHTML filter in .eleventy.js #}
```

Comment:
I'm not proud of this, but I think it'd be a disproportionate amount of work to find a cleaner approach